.config/dotnet-tools.json

```diff
@@ -1,18 +0,0 @@
-{
-  "version": 1,
-  "isRoot": true,
-  "tools": {
-    "dotnet-reportgenerator-globaltool": {
-      "version": "4.6.4",
-      "commands": [
-        "reportgenerator"
-      ]
-    },
-    "nbgv": {
```

Comment:
How will we manage the template version without nbgv? The builds are now producing packages versioned 0.0.0

Reply:
I know, this PR isn't ready for review yet. I planned to restore nbgv, just didn't get to it yet. I only opened this PR to verify cibuild works.

Reply:
This working again now.